### ISSUE TITLE
cog errors appear in apache log file on data only nodes

### ISSUE THREAD

**Issue by ncarenton** [ESGF/apache-frontend#5]
Hi,

We should probably have a data only node specific config for apache or we will find these errors in the log:

 Target WSGI script not found or unable to stat: /usr/local/cog

**Comment by LucaCinquini** [comment]
Yes. One other change is that for a data node only , the base URL / should redirect to threads/catalog.html
thanks, L
